src/main/java/com/google/cloud/spanner/jdbc/JdbcStatement.java

```diff
@@ -87,7 +87,7 @@ public long executeLargeUpdate(String sql) throws SQLException {
     switch (result.getResultType()) {
       case RESULT_SET:
         throw JdbcSqlExceptionFactory.of(
-            "The statement is not an update or DDL statement", Code.INVALID_ARGUMENT);
+            "The statement is not a normal update or DDL statement", Code.INVALID_ARGUMENT);
```

Comment:
What does normal mean? Non-returning?

Reply:
Yes, right. I think mentioning this as non-returning makes more sense here. Will change it.

Reply:
Changed.